contributing_to_docs/doc_guidelines.adoc

```diff
@@ -828,6 +828,55 @@ Embedding external files is restricted for files that change frequently, like te
 You must get approval from the Engineering, QE, and Docs teams before embedding an external file.
 ====
 
+== Embedding a local YAML file
+
```

Comment:
You might want to add a caveat about [line specifications in the include directive](https://docs.asciidoctor.org/asciidoc/latest/directives/include-lines/). They should be used with caution because they might need to be updated if someone changes the YAML file. We have no easy way to trace where a snippet is used, besides searching the repo, and maintaining line ranges would probably outweigh the advantages of having a YAML snippet.

However, a YAML snippet would be useful if the snippet contained only the "common" lines and then add the "differences" in the code block of the module.

Reply:
I suggest that we recommend against using line specs for "snippets" of included files. @kalexand-rh WDYT?

Reply:
Guess this https://github.com/openshift/openshift-docs/pull/60315 addresses it

Reply:
I think it would be fair to cover it in both places.

Reply:
Added a note in my PR, PTAL

Reply:
I think that looks good!